R/tmb-methods.R

```diff
@@ -191,35 +193,170 @@ h_get_prediction_variance <- function(object, n_sim, tmb_data) {
 #' # Model frame:
 #' model.frame(object)
 #' model.frame(object, include = "subject_var")
-model.frame.mmrm_tmb <- function(formula, data, include = NULL, full, na.action = "na.omit", ...) { # nolint
-  include_choice <- c("subject_var", "visit_var", "group_var", "response_var")
+model.frame.mmrm_tmb <- function(formula, data, include = c("subject_var", "visit_var", "group_var", "response_var"),
+                                 full, na.action = "na.omit", ...) { # nolint
+  # Construct updated formula and data arguments.
+  lst_formula_and_data <-
+    h_construct_model_frame_inputs(
+      formula = formula,
+      data = data,
+      include = include,
+      full = full
+    )
+
+  # construct data frame to return
```

Comment:
Please sentence style for all comments, thx :)

Reply:
updated throughout